Fails to install on Ubuntu 18.04 wit ruby 2.5

```
sudo gem install stringio
Building native extensions. This could take a while...
ERROR:  Error installing stringio:
	ERROR: Failed to build gem native extension.

    current directory: /var/lib/gems/2.5.0/gems/stringio-0.0.1
/usr/bin/ruby2.5 -r ./siteconf20180424-25285-1aa9ae4.rb extconf.rb
creating Makefile

current directory: /var/lib/gems/2.5.0/gems/stringio-0.0.1
make "DESTDIR=" clean

current directory: /var/lib/gems/2.5.0/gems/stringio-0.0.1
make "DESTDIR="
make: *** No rule to make target '/include/ruby.h', needed by 'stringio.o'.  Stop.

make failed, exit code 2

Gem files will remain installed in /var/lib/gems/2.5.0/gems/stringio-0.0.1 for inspection.
Results logged to /var/lib/gems/2.5.0/extensions/x86_64-linux/2.5.0/stringio-0.0.1/gem_make.out
```

--- a/stringio.gemspec
+++ b/stringio.gemspec
@@ -12,8 +12,8 @@ Gem::Specification.new do |s|
   s.authors = ["Nobu Nakada".freeze]
   s.description = "Pseudo `IO` class from/to `String`.".freeze
   s.email = "nobu@ruby-lang.org".freeze
-  s.extensions = ["extconf.rb".freeze]
-  s.files = ["README.md".freeze, "depend".freeze, "extconf.rb".freeze, "stringio.c".freeze]
+  s.extensions = ["ext/stringio/extconf.rb".freeze]
+  s.files = ["README.md".freeze, "ext/stringio/extconf.rb".freeze, "ext/stringio/stringio.c".freeze]
   s.homepage = "https://github.com/ruby/stringio".freeze
   s.licenses = ["BSD-2-Clause".freeze]
   s.required_ruby_version = Gem::Requirement.new(">= 2.2".freeze)

